validator/sawtooth_validator/metrics/metrics.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import os
+import platform
+
+
+class MetricsConfig:
+    DEBUG = 10
+    INFO = 20
+    PERF = 30
+
+    def __init__(self, reporting_level):
```

Comment:
Alternatively, I don't see this being used as a class, so it could just be an `IntEnum`

Reply:
As I mentioned above, this will be extended to include more information about which modules and classes are enabled/disabed, not just the reporting level.

Reply:
That makes this class premature then. It should be part of that future PR.